src/Databases/DatabaseAtomic.cpp

```diff
@@ -518,8 +528,8 @@ LoadTaskPtr DatabaseAtomic::startupDatabaseAsync(AsyncLoader & async_loader, Loa
                 std::lock_guard lock{mutex};
                 table_names = table_name_to_path;
             }
-
-            fs::create_directories(path_to_table_symlinks);
+            if (db_disk->isSymlinkSupported())
```

Comment:
Also skip the loop below in this case?

Reply:
I will remove it.

Reply:
The logic in the loop has an effect to ensure that all tables are loaded. So I keep it unchanged. In `tryCreateSymlink`, I already skipped the logic if the symlink is not supported.